README.md

```diff
@@ -0,0 +1,69 @@
+Configuration files for Intel AudioVoiceSpeech (AVS) driver
+========================
+
+ * [Overview](#overview)
+   * [DMIC](#dmic)
+   * [HDAudio](#hdaudio)
+   * [HDMI](#hdmi)
+   * [I2S](#i2s)
+
+# Overview
+
+Each configuration file represents an audio stream topology i.e.: connections of AudioDSP pipelines
+and processing modules that provide rich user experience with the Intel AVS stack. The avs-driver is
+part of the Linux kernel sound subsystem, `sound/soc/intel/avs`. If you do not possess clone of
+kernel's tree, you can view it by visiting
+[Bootlin](https://elixir.bootlin.com/linux/latest/source/sound/soc/intel/avs).
+
+Mimics [alsa-topology-conf](https://github.com/alsa-project/alsa-topology-conf) yet instead
+of UseCaseManagement (UCM) files, topologies found here are XML document files.
+
+As the audio stack supports audio transfer over multiple interfaces, repo consists of directories
```

Comment:
Use full words, like "repository" instead of "repo"?

Reply:
Agreed.